params/config.go

```diff
@@ -207,6 +209,14 @@ func (c *ChainConfig) String() string {
 	)
 }
 
+func (c *ChainConfig) IsValid() error {
+	if c.SizeLimit < 32 || c.SizeLimit > 128 {
```

Comment:
TODO: Determine sane sizes

Reply:
Quorum block gas limit is set to be 3,758,096,384 and Quorum is gas-less, so the txn gas price is not accounted for. This means, that about 6MB of data fits into the block, which in turns means that our maximum transaction size limit could theoretically be up to 6000KB

Reply:
Leaving at 128 until a reasonable request for a higher limit comes up.